operator/pkg/ciliumenvoyconfig/envoy_config.go

```diff
@@ -258,7 +258,9 @@ func getConnectionManager(svc *slim_corev1.Service) (ciliumv2.XDSResource, error
 		},
 	}
 
-	var mutatorFuncs = []httpConnectionManagerMutator{}
+	var mutatorFuncs = []httpConnectionManagerMutator{
+		grpcHttpConnectionManagerMutator(svc),
```

Comment:
it's related to above comment from Chance. We are not configuring auto upgrade for websocket, hence we don't need to check if it's grpc anymore.

Agreed that input for this function is not used, this is just to keep the same mutator function signature, so that we can add more features later.